### PR TITLE
add link to logo

### PR DESCRIPTION
**Pull request by Philip-A-R**
Hi @trentgillis ok, so I added a clickable link (that works). But I don't know where to actually link it to, so I just linked it to google. How do I link it to the root index page? 

Thanks!

### CODE REVIEW

File: components/Logo/Logo.tsx [hunk 1]
```diff
@@ -17,7 +18,9 @@ const LogoWrapper = styled.div`
 export function Logo() {
   return (
     <LogoWrapper>
-      <Image src={logo} alt="logo" fill priority sizes="100vh" />
+      <Link href="https://www.google.com/">
```
Comment (trentgillis) [inline components/Logo/Logo.tsx:21]: `href="/"` should work for returning users to the root index page

**Comment by trentgillis** [comment]
@Philip-A-R you'll want to tag the issue as closed through the PR description. To do that doe `close #X` where X is the issue number

File: components/Logo/Logo.tsx (hunk 1 above)
Comment (Philip-A-R) [reply components/Logo/Logo.tsx]: Done!

**Review by trentgillis — APPROVED** [review]
LGTM (looks good to me)

**Pull request merged**